### DISCUSSION TITLE
Click on "New discussion" to start discussion.

### DISCUSSION THREAD

**Discussion by tr3v3r** [tr3v3r/react-native-esc-pos-printer#163] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started,  click on the "New Discussion" button under the corresponding section.

**Comment by rastakorps** [comment]
One question, currently printing via USB is working on android?

**Reply by tr3v3r** [reply]
[Check https://github.com/tr3v3r/react-native-esc-pos-printer/blob/main/docs/printer/constructor.md](https://github.com/tr3v3r/react-native-esc-pos-printer/blob/main/docs/printer/constructor.md)

**Comment by sarmad-321** [comment]
Hi tr3v3r. i wanted to ask after this new implementation there is a function printerInstance.addCommand() what does it do ? 
also i want to know i had implementation on web where i used to create a complete series of ascii code to print. so can i use that same string over here ? 
Thanks.

**Reply by tr3v3r** [reply]
@sarmad-321  please add new questions here

<img width="1287" alt="image" src="https://github.com/user-attachments/assets/c3da6a1c-1b93-4388-8c55-d65f3ca774e4">
by clicking on "new discussion"
